src/main/java/decodex/modules/hex/HexEncoder.java

```diff
@@ -33,4 +32,5 @@ public Data run(Data data) {
         String encodedString = encodedStringBuilder.toString().trim();
         return new Data(encodedString);
```

Comment:
Perhaps not do a `trim()`? This may accidentally lose some data. (e.g the user may intend to keep the trailing whitespaces)